translations/en.json

```diff
@@ -519,8 +519,9 @@
   "record.metadata.title": "Title",
   "record.metadata.topics": "Categories",
   "record.metadata.uniqueId": "Unique Identifier",
+  "record.metadata.update": "Date of update",
   "record.metadata.updateFrequency": "Data Update Frequency",
-  "record.metadata.updatedOn": "Updated on",
+  "record.metadata.updatedOn": "Description modified on",
```

Comment:
```suggestion
  "record.metadata.updatedOn": "Metadata modified on",
```
Even though we want to limit usage of the term "metadata", I think it would make sense here to not confuse "Description" with the record's abstract. Using "metadata" would also be consistent with the line below:
<img width="270" height="116" alt="image" src="https://github.com/user-attachments/assets/d688a87f-d053-4a33-bc99-4f0f875223d9" />